docs/elevation.md

```diff
@@ -1,39 +0,0 @@
-Elevation Lookup Service
----------
-This document describes how to run an elevation lookup service using Valhalla. Documentation for this service can be found here: [elevation-api-docdocs](https://github.com/valhalla/valhalla-docs/tree/master/elevation/elevation-service.md). The easiest way to start an elevation service is to install Valhalla from PPA, download elevation data, and start the service. If you prefer to buld Valhalla from source please refer to the Valhalla [README](../README.md).
-
-Get Valhalla from Personal Package Archive (PPA)
-------------------------------------------------
-If you are running Ubuntu (trusty or xenial) Valhalla can be installed quickly and easily via PPA. Try the following:
-
-```bash
-# grab all of the valhalla software from ppa
-sudo add-apt-repository -y ppa:valhalla-core/valhalla
-sudo apt-get update
-sudo apt-get install -y valhalla-bin
-```
-
-Running
--------
-
-The following bash should be enough to make some get some elevation data and start a server using it:
-
-```bash
-# grab data for the whole world (its about 1.6TB) or a smaller bounding box
-valhalla_build_elevation -180 180 -90 90 ./elevation_tiles $(nproc)
-#configure the server
-valhalla_build_config --additional-data-elevation ./elevation_tiles > config.json
-#start up the server with the config and number of threads
-valhalla_elevation_service config.json 1
```

Comment:
this file should not be deleted IMHO. all of it is still correct. the only thing that needs to be updated is to change this line here to be: `valhalla_service config.json 1`